### PR TITLE
Upgrade to 0.8.0

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,8 +1,14 @@
 {
   "name": "paper-progress",
+  "version": "0.8.0",
   "private": true,
```
Comment (cdata) [inline bower.json:4]: This element needs the following fields in `bower.json`: `"main"` and `"authors"`

File: bower.json [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
   "dependencies": {
-    "core-range": "Polymer/core-range#master",
-    "paper-button": "Polymer/paper-button#master"
+    "iron-range-behavior": "PolymerElements/iron-range-behavior#0.8"
+  },
+  "devDependencies": {
+    "iron-component-page": "PolymerElements/iron-component-page#^0.8.0",
+    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.6.0",
+    "paper-button": "PolymerElements/paper-button#^0.8"
```
Comment (cdata) [inline bower.json]: `0.8.0`

File: index.html [hunk 3]
```diff
@@ -9,14 +10,15 @@
 -->
 <html>
 <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (cdata) [inline index.html]: Use the `meta` from here: https://github.com/PolymerElements/paper-progress/pull/1/files#diff-3e22c50371c76d64ea9738dbc46a6e57R15

File: demo/index.html [hunk 4]
```diff
@@ -1,84 +1,84 @@
...
       display: block;
       width: 100%;
       padding: 25px 0;
     }
-    
-    paper-progress.pink::shadow #activeProgress {
-      background-color: #e91e63;
+
+    paper-progress.pink {
+     --paper-progress-active-color: #e91e63;
```
Comment (cdata) [inline demo/index.html]: Indentation

File: paper-progress.html [hunk 5]
```diff
@@ -27,98 +28,169 @@
...
+    observers: [
+      '_updateProgress(value, min, max, indeterminate)',
+      '_ratioChanged(ratio)',
+      '_secondaryProgressChanged(secondaryProgress, min, max)'
+    ],
+
+    _updateProgress: function() {
+      // If we use attribute/class binding, the animation sometimes doesn't translate properly
+      // on Safari 7.1. So instead, we toggle the class here in the update method.
+      this.$.activeProgress.classList.toggle('indeterminate', this.indeterminate);
```
Comment (cdata) [inline paper-progress.html:175]: This doesn't seem to actually use the other dependent values (`value`, `min` or `max`) for anything. Are they needed?

Comment (blasten) [inline paper-progress.html:175]: that's right. I actually changed that function a little bit.

**Comment by cdata** [comment]
Will this still be merged? If not, can we close it?

**Comment by blasten** [comment]
Yeah.. I think we can close it.

**Pull request closed without merging**